UI/data/locale/en-US.ini

```diff
@@ -867,6 +867,10 @@ Basic.Settings.Stream.Recommended.MaxVideoBitrate="Maximum Video Bitrate: %1 kbp
 Basic.Settings.Stream.Recommended.MaxAudioBitrate="Maximum Audio Bitrate: %1 kbps"
 Basic.Settings.Stream.Recommended.MaxResolution="Maximum Resolution: %1"
 Basic.Settings.Stream.Recommended.MaxFPS="Maximum FPS: %1"
+Basic.Settings.Stream.ChangeIncompatibleEncoder.Title="Incompatitle encoder changed"
+Basic.Settings.Stream.ChangeIncompatibleEncoder.Simple="The actually selected encoder is not compatible with the actual service.\nThe encoder in the output page was changed by its H264 version."
```

Comment:
Not sure about "The actually selected encoder". Is this distinct somehow from "The selected encoder"? Same with "the actual service" vs. "the service".